LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
i think we also need to add license to each file

Reply:
I will file an issue for it. I think we could delay it since we are not sure if we will donate the project to Kubeflow community.